app/src/main/java/asd/org/ahorcado/dal/DataBaseHelper.java

```diff
@@ -0,0 +1,115 @@
+package asd.org.ahorcado.dal;
+
+import android.content.Context;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteException;
+import android.database.sqlite.SQLiteOpenHelper;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.sql.SQLException;
+
+import asd.org.ahorcado.BuildConfig;
+
+/**
+ * Created by Huber on 17/04/2016.
+ */
+public class DataBaseHelper extends SQLiteOpenHelper {
+    private Context mycontext;
+    private static String DB_NAME = "Ahorcado.db";
+    private static String DB_PATH ="/data/data/"+ BuildConfig.APPLICATION_ID+"/databases/";
+    public SQLiteDatabase myDataBase;
+
+    public DataBaseHelper(Context context) throws IOException {
+        super(context,DB_NAME,null,1);
+        this.mycontext=context;
+        boolean dbexist = checkdatabase();
+        if (dbexist) {
+            System.out.println("Database exists");
+            try {
+                opendatabase();
+            } catch (SQLException e) {
+                e.printStackTrace();
+            }
+        } else {
+            System.out.println("Database doesn't exist");
+            createdatabase();
+        }
+    }
+
+    public void createdatabase() throws IOException {
+        boolean dbexist = checkdatabase();
+        if(dbexist) {
+            System.out.println(" Database exists.");
+        } else {
+            this.getReadableDatabase();
+            try {
+                copydatabase();
+            } catch(IOException e) {
+                throw new Error("Error copying database");
+            }
+        }
+    }
+
+    private boolean checkdatabase() {
+
+        boolean checkdb = false;
+        try {
+            String myPath = DB_PATH + DB_NAME;
+            File dbfile = new File(myPath);
+            checkdb = dbfile.exists();
+        } catch(SQLiteException e) {
+            System.out.println("Database doesn't exist");
+        }
+        return checkdb;
+    }
+
+    private void copydatabase() throws IOException {
+        //Open your local db as the input stream
+        InputStream myinput = mycontext.getAssets().open(DB_NAME);
+
+        // Path to the just created empty db
+        String outfilename = DB_PATH + DB_NAME;
```

Comment:
If we use constants to define the path, this concatenation could be in other method